apis/keda/v1alpha1/triggerauthentication_types.go

```diff
@@ -232,6 +241,31 @@ type HashiCorpVault struct {
 	Mount string `json:"mount,omitempty"`
 }
 
+// Akeyless is used to authenticate using Akeyless
+type Akeyless struct {
+	GatewayURL string           `json:"gatewayUrl"`
+	AccessID   string           `json:"accessId"`
+	Secrets    []AkeylessSecret `json:"secrets"`
+
+	// +optional
+	AccessKey *string `json:"accessKey,omitempty"`
```

Comment:
This looks as a plain text key, we must avoid adding secrets directly into KEDA CRDs, please use a reference from a secret or so